scripts/create-nuxt-app/pr.md

```diff
@@ -9,6 +9,16 @@
    4. Add [pages/index.spec.ct.js](pages/index.spec.ct.js), [components/Logo.spec.ct.js](components/Logo.spec.ct.js)
    5. `npx cypress open-ct`
    6. ${COMMIT_1}
+3. Add Cypress Code Coverage
```

Comment:
There are a few issues with this section:

- The `cypress/support/commands.js` path is incorrect
- The `src/index.spec.ct.js` path is incorrect
- There is no `components/Counter.spec.ct.js` file
- There are other files added in this step that are not accounted for in this bulleted list

Also, we may also want to explain why we're adding the `.nycrc` file (eg. "to configure Istanbul")

Are we adding the `Counter` component just to test code coverage? Because it might make more sense to add this component and test as part of the "Add Cypress with example component and page tests" step.